pkg/os/util.go

```diff
@@ -113,3 +113,17 @@ func RemoveFileIfExists(path string) error {
 func RunningUsingSSH() bool {
 	return os.Getenv("SSH_TTY") != ""
 }
+
+// RemoveFileGlob takes a glob pattern as string and removes the files that matches
```

Comment:
`the files and directories that matches` (because of the use of `RemoveAll`)

Reply:
@cfergeau @praveenkumar after this change we can merge, right?

@anjannath ^^^

Reply:
updated